### DISCUSSION TITLE
Neovim Copilot should accept an auth token

### DISCUSSION THREAD

**Discussion by TravisDart** [community/community#127418] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Since neovim runs in the terminal, some environments don't have a browser available for the `:Copilot setup` authentication. The `gh` CLI program will accept a GitHub token for auth, and it would be convenient if the Neovim Copilot could auth with a token, as well. I looked at the [source code](https://github.com/github/copilot.vim/blob/release/autoload/copilot.vim#L641), and currently, there's no provision for tokens.

**Comment by magnusriga** [comment]
Did you find an answer here, @TravisDart ? I am running nvim in a dev container with no direct browser access.
